### PR TITLE
Make it possible and convenient to run the host orchestrator in a user workstation

### PR DESCRIPTION
**Pull request by jemoreira**
Changed default values and added the command to the debian package.

### CODE REVIEW

File: frontend/debian/cuttlefish-orchestration.cuttlefish-host_orchestrator.init [hunk 1]
```diff
@@ -79,13 +79,34 @@ start() {
...
   chown _cutf-operator:cvdnetwork "${orchestrator_cvd_artifacts_dir}"
 
-  eval $(set_config_expr ORCHESTRATOR_HTTP_PORT orchestrator_http_port)
-  eval $(set_config_expr ORCHESTRATOR_HTTPS_PORT orchestrator_https_port)
-  eval $(set_config_expr ORCHESTRATOR_TLS_CERT_DIR orchestrator_tls_cert_dir)
-  eval $(set_config_expr ORCHESTRATOR_ANDROID_BUILD_URL orchestrator_android_build_url)
-  eval $(set_config_expr ORCHESTRATOR_CVDBIN_ANDROID_BUILD_ID orchestrator_cvdbin_android_build_id)
-  eval $(set_config_expr ORCHESTRATOR_CVDBIN_ANDROID_BUILD_TARGET orchestrator_cvdbin_android_build_target)
-  eval $(set_config_expr ORCHESTRATOR_CVD_ARTIFACTS_DIR orchestrator_cvd_artifacts_dir)
+  orhcestrator_args=()
```
Comment (ser-io) [inline frontend/debian/cuttlefish-orchestration.cuttlefish-host_orchestrator.init:82]: can this block be moved into its own function aiming for a shorter `start` function?

File: frontend/src/host_orchestrator/main.go [hunk 2]
```diff
@@ -81,45 +81,48 @@ func start(starters []func() error) {
...
+
+	certPath := filepath.Join(*tlsCertDir, "cert.pem")
+	keyPath := filepath.Join(*tlsCertDir, "key.pem")
+
 	imPaths := orchestrator.IMPaths{
-		RootDir:          imRootDir,
-		CVDToolsDir:      imRootDir,
-		ArtifactsRootDir: filepath.Join(imRootDir, "artifacts"),
-		RuntimesRootDir:  filepath.Join(imRootDir, "runtimes"),
+		RootDir:          *imRootDir,
```
Comment (ser-io) [inline frontend/src/host_orchestrator/main.go:126]: nit: using the Var() functions we can avoid the `*` resulting in a more consistent reading/view of the code.

File: frontend/src/host_orchestrator/main.go [hunk 3]
```diff
@@ -38,10 +39,17 @@ const (
 	defaultAndroidBuildURL          = "https://androidbuildinternal.googleapis.com"
 	defaultCVDBinAndroidBuildID     = "10796991"
 	defaultCVDBinAndroidBuildTarget = "aosp_cf_x86_64_phone-trunk_staging-userdebug"
-	defaultCVDArtifactsDir          = "/var/lib/cuttlefish-common"
 	DefaultListenAddress            = "127.0.0.1"
 )
 
+func defaultCVDArtifactsDir() string {
+	u, e := user.Current()
```
Comment (ser-io) [inline frontend/src/host_orchestrator/main.go:46]: nit: `err` for consistency.

**Review by ser-io — APPROVED** [review]

File: frontend/debian/cuttlefish-orchestration.cuttlefish-host_orchestrator.init (hunk 1 above)
Comment (jemoreira) [reply frontend/debian/cuttlefish-orchestration.cuttlefish-host_orchestrator.init:103]: Done

File: frontend/src/host_orchestrator/main.go (hunk 3 above)
Comment (jemoreira) [reply frontend/src/host_orchestrator/main.go]: Done

File: frontend/src/host_orchestrator/main.go (hunk 2 above)
Comment (jemoreira) [reply frontend/src/host_orchestrator/main.go:126]: discussed offline, keeping like this for now

**Pull request merged**